docs/note-new-orders-with-their-fulfillment-locations/README.md

```diff
@@ -0,0 +1,35 @@
+# Add note to new orders with their fulfillment locations
+
+Tags: Fulfillment, Location, Orders
+
+Monitoring for brand-new orders, this task add a note to each order with the fulfillment locations that Shopify automatically assigns at the moment of order creation.
+
+* View in the task library: [tasks.mechanic.dev/note-new-orders-with-their-fulfillment-locations](https://tasks.mechanic.dev/note-new-orders-with-their-fulfillment-locations)
+* Task JSON, for direct import: [task.json](../../tasks/note-new-orders-with-their-fulfillment-locations.json)
+* Preview task code: [script.liquid](./script.liquid)
+
+## Subscriptions
+
+```liquid
+shopify/orders/create
+```
+
+[Learn about event subscriptions in Mechanic](https://learn.mechanic.dev/core/tasks/subscriptions)
+
+## Documentation
+
+Monitoring for brand-new orders, this task add a note to each order with the fulfillment locations that Shopify automatically assigns at the moment of order creation.
+
+Important: This task requires Shopify API version 2020-01 or later.
```

Comment:
I believe this note can be removed (I'm aware this is a vestige of the original task you based this new one on).

A task cannot be configured with an API version beyond the last supported Shopify API version (4 quarters prior) and Mechanic will auto bump task API versions when they are getting close to the unsupported date.